packages/firebase_ml/android/src/main/java/io/flutter/plugins/firebaseml/FirebaseMLPlugin.java

```diff
@@ -40,21 +50,86 @@ public void onAttachedToEngine(@NonNull FlutterPluginBinding flutterPluginBindin
     channel.setMethodCallHandler(this);
   }
 
+  @RequiresApi(api = Build.VERSION_CODES.N)
   @Override
-  public void onMethodCall(@NonNull MethodCall call, @NonNull Result result) {
+  public void onMethodCall(@NonNull MethodCall call, @NonNull final Result result) {
+    final FirebaseCustomRemoteModel remoteModel;
+    String modelName;
```

Comment:
Since these are not used in the wider scope it's preferred to declare separate local variables in limited scope as they are needed.

https://screenshot.googleplex.com/gTaOBnWeu8h
go/java-style#s4.8.2.2-variables-limited-scope

Reply:
Done